lang/en.json

```diff
@@ -146,12 +146,16 @@
 "DND5E.AddEmbeddedItemPromptHint": "Do you want to add these items to your character sheet?",
 "DND5E.AdvancementAbilityScoreImprovementTitle": "Ability Score Improvement",
 "DND5E.AdvancementAbilityScoreImprovementHint": "Allow the player to increase one or more ability scores or take an optional feat.",
+"DND5E.AdvancementAbilityScoreImprovementCap": "Point Cap",
+"DND5E.AdvancementAbilityScoreImprovementCapHint": "Maximum number of points a player can assign to a single score.",
+"DND5E.AdvancementAbilityScoreImprovementCapDisplay.one": "Max {points} point per score",
+"DND5E.AdvancementAbilityScoreImprovementCapDisplay.other": "Max {points} points per score",
 "DND5E.AdvancementAbilityScoreImprovementFeatHint": "Drop a feat here to choose one instead of an ability score improvement.",
 "DND5E.AdvancementAbilityScoreImprovementFeatWarning": "Only features with the 'feat' type can be selected.",
 "DND5E.AdvancementAbilityScoreImprovementFixed": "Fixed Improvement",
 "DND5E.AdvancementAbilityScoreImprovementLockedHint": "Scores cannot be modified with a feat selected.",
 "DND5E.AdvancementAbilityScoreImprovementPoints": "Points",
-"DND5E.AdvancementAbilityScoreImprovementPointsHint": "Number of points that can be assigned to any ability score below the maximum that doesn't have a fixed improvement set.",
+"DND5E.AdvancementAbilityScoreImprovementPointsHint": "Number of points that can be assigned to any ability score below the maximum that doesn't have a fixed improvement set. Maximum points per-score will be limited by point cap if set.",
```

Comment:
This was a bit of a mouthful to read when I had a look at the configuration sheet. I'd recommend omitting the last sentence on maximum points and going with the hint that we have added a few lines earlier. Maybe that was the intention and this last sentence was accidentally left in?

I might also suggest removing the 'below the maximum' as that should be implied from the general game rules, but I don't feel particularly strongly.